.github/workflows/build.yml

```diff
@@ -94,6 +94,7 @@ jobs:
         run: ./gradlew check --stacktrace --info
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_USERNAME: jensim
```

Comment:
Maybe we could use GITHUB_USERNAME: ${{ GITHUB_ACTOR }} or whatever its called?
Would be easier for people who fork?

Reply:
Done